app/src/Store.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App;
+
+Use App\Book as Book;
+
+/**
+ * Class Store - prepare and execute query to DB to get all book names, ids, prices and images
+ * @package App
+ * @return array(object) of all books
+ */
+class Store
+{
+    public $db;
+
+    /**
+     * Store constructor.
+     * @param DbConnector $connection
+     */
+    public function __construct(DbConnector $connection)
+    {
+        $this->db = $connection->db;
+    }
+
+    /**
+     * gets all the books and their data from DataBase
+     * @return array of book objects
+     */
+    public function getAllBooks()
+    {
+        $query = $this->db->prepare("SELECT `id`, `title`, `price`, `image` FROM `books`;");
+        $query->setFetchMode(\PDO::FETCH_CLASS, Book::class);
+        $query->execute();
+        $books = $query->fetchAll();
+        return $books;
+    }
+
+    /**
+     * @param $books - returned from getAllBooks, an array of objects
+     */
+    public function showAllBooks($books)
+    {
+        foreach ($books as $book) {
+            echo "<div class='listedBook col-xs-4'>
```

Comment:
Please can this string be returned here and then echoed when it is called in the index.php file.